core-bundle/src/Resources/contao/dca/tl_layout.php

```diff
@@ -507,7 +507,7 @@ public function checkPermission()
 	 */
 	public function getStyleSheets(DataContainer $dc)
 	{
-		$intPid = $dc->activeRecord->pid;
+		$intPid = $dc->activeRecord->pid ?? null;
```

Comment:
I would use `0` instead of `null` to generate a valid SQL query

Reply:
We have agreed on always using `?? null`, because anything else like `?? ''` or `?? false` is a behavior change.